bin/eslint-github-init.js

```diff
@@ -68,14 +72,59 @@ inquirer.prompt(questions).then(answers => {
     eslintrc.extends.push('plugin:github/browser')
   }
 
-  if (answers.flow) eslintrc.extends.push('plugin:github/flow')
+  if (answers.typeSystem === 'flow') eslintrc.extends.push('plugin:github/flow')
+  if (answers.typeSystem === 'typescript') {
+    eslintrc.extends.push('plugin:github/typescript')
+
+    // Create a `tsconfig.json` if one doesn't exits. Equivalent to running `tsc --init`.
+    const tsconfigPath = path.resolve(process.cwd(), 'tsconfig.json')
+    if (!fs.existsSync(tsconfigPath)) {
```

Comment:
It might be, I'm not too sure about this but we might have to make sure that it is configured in a certain way. We could of course just skip this and document it instead.